Tools/MGCB/CommandLineParser.cs

```diff
@@ -284,6 +284,8 @@ private IEnumerable<string> Preprocess(IEnumerable<string> args)
                 if (arg.StartsWith("/@:") || arg.StartsWith("-@:"))
                 {
                     var file = arg.Substring(3);
+                    Directory.SetCurrentDirectory(Path.GetDirectoryName(file));
```

Comment:
I'm nervous at this sort of change.  It has the potential to break every other file operation.

First why was this not a problem before?  We've been using MGCB on Mac/Linux for months now...  at least a year on Windows.  What is different about your case and what happens to all the previously working cases?